static/app/views/settings/organizationMembers/organizationMemberDetail.tsx

```diff
@@ -383,10 +384,11 @@ class OrganizationMemberDetail extends DeprecatedAsyncView<Props, State> {
         <OrganizationRoleSelect
           enforceAllowed={false}
           enforceRetired={this.hasTeamRoles}
-          disabled={!canEdit}
+          disabled={!canEdit || isPartnershipUser}
           roleList={orgRoleList}
           roleSelected={orgRole}
           setSelected={this.onChangeOrgRole}
+          helpText={isPartnershipUser ? t('This is a partnership user.') : undefined}
```

Comment:
i need some text suggestions that aren't too long

Reply:
cc: @Dhrumil-Sentry 
I'm really bad at this. How about `Partnership users can't be modified`?

Reply:
1. `Cannot change the role of partnership users`
2. `It's not possible to alter the partnership users' roles.`
3. `Modifying the roles of partnership users is not an option.`
4. `Partnership users' roles are fixed and cannot be changed.`
5. `The roles of partnership users are unmodifiable.`
6. `There is no way to adjust the partnership users' roles.`